dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/WECC/package.order

```diff
@@ -0,0 +1,6 @@
+BaseControls
```

Comment:
I don't get why we don't have the same tree structure for controls as for models? Photovoltaics before WECC folder?

Reply:
In Models we have Photovoltaics then WECC.
In controls we have WECC and nothing specific to Photovoltaics